### ISSUE TITLE
How to apply wdio config?

### ISSUE THREAD

**Issue by adamduffy** [FormidableLabs/rowdy#49]
I want to set the global waitforTimeout on webdriverio.  http://webdriver.io/guide/testrunner/timeouts.html#WebdriverIO_related_timeouts

how is this done in rowdy config?  (i am also using sauceLabs)

**Comment by ryan-roemer** [comment]
@adamduffy -- Reading the wdio source, I think it's going to be a configuration key that goes right here: https://github.com/FormidableLabs/rowdy/blob/master/config.js#L168-L177

If you can paste your code snippet of how you are initializing rowdy with a custom configuration, I can probably show you how to tweak it to insert the global config value...

**Comment by adamduffy** [comment]
that seems to have worked!  thanks.   not where i would have expected it to be though.

**Comment by ryan-roemer** [comment]
Yeah, that's basically the object passed to construct the wdio object...

If you get a second and want to enhance the README.md or paste a snippet of code in this issue for posterity's sake, might be nice for the next developer that comes along trying to do the same thing!

**Comment by adamduffy** [comment]
now i'm also unable to figure out how to set `waitforTimeout` on the local config
